app/services/idv/steps/doc_auth_base_step.rb

```diff
@@ -156,6 +157,16 @@ def verify_document_capture_session_uuid_key
         :verify_document_action_document_capture_session_uuid
       end
 
+      def track_document_state
+        return unless IdentityConfig.store.state_tracking_enabled
+        state = flow_session[:pii_from_doc][:state]
+        return unless state
+        doc_auth_log = DocAuthLog.find_by(user_id: user_id)
+        return unless doc_auth_log
```

Comment:
Is there only one DocAuthLog per state? Would we ever want historical data, like if a user moves or tries an ID from a different state?

Reply:
Yes there is only one doc_auth_log per user.  It would contain the last state they tried